src/effects/browse.js

```diff
@@ -12,12 +12,15 @@ export function fetchSchools() {
     );
 }
 
-export function fetchSubjects(schoolId) {
+export function fetchSubjects(termId, schoolId) {
```

Comment:
I thought we got rid of this?

Reply:
I thought I had resolved the merge conflicts too. I'm not quite sure why this appeared, but I'll fix it